src/core.jl

```diff
@@ -1370,15 +1370,37 @@ Base.haskey(graph::Graph, name) = isnull(get_node_by_name(graph, name))
 
 
 
+node_name(::Void) = nothing
+node_name(xs::AbstractVector)=node_name.(xs)
```

Comment:
Yes, this would only work on 0.6.

It may be a good idea to drop 0.5 fairly quickly, but that's up to you / Jon.